python_modules/libraries/dagster-k8s/dagster_k8s/ops/k8s_job_op.py

```diff
@@ -203,7 +203,7 @@ def execute_k8s_job(
         else None,
     )
 
-    container_config = container_config or {}
+    container_config = container_config.copy() or {}
```

Comment:
should we update the mypy type to `Mapping` to indicate it is / can be read only?

Reply:
I like this idea but it doesn't like calling .copy() on a Mapping

Reply:
(mypy doesn't)

Reply:
To do this with Mapping you can use unpack it using `**`.

`new_dict = {**mapping}`